src/snowflake/snowpark/dataframe.py

```diff
@@ -543,6 +587,14 @@ def __init__(
 
         self._alias: Optional[str] = None
 
+    def _set_ast_ref(self, sp_dataframe_expr_builder: Any) -> None:
+        """
+        Given a field builder expression of the AST type SpDataframeExpr, points the builder to reference this dataframe.
+        """
+        # TODO: remove once we generate the correct AST.
```

Comment:
can we add JIRA with this TODO

Reply:
added! cf. https://github.com/snowflakedb/snowpark-python/pull/2506